lua/bufferline/render.lua

```diff
@@ -167,23 +177,19 @@ local function render(update_names)
 
     local padding = string.rep(' ', layout.padding_width)
 
-    local width =
-      buffer_data.width ~= nil and
-        buffer_data.width or
-        (layout.base_widths[i] + 2 * layout.padding_width)
-
     local item = {
-      width = width,
-      groups = {
+      width = buffer_data.width
+        or (layout.padding_width + layout.base_widths[i] + layout.padding_width),
```

Comment:
I don't find this clearer than `2 * layout.padding_width`. Maybe it's just my time as a frontend dev but for me "padding" always means "space applied to both sides".